ard_controller/ard_controller.ino

```diff
@@ -0,0 +1,29 @@
+#include <DueTimer.h>
+
+bool write_flag = false;
+
+void setup() {
+  // put your setup code here, to run once:
+  Serial.begin(115200);
+  Serial1.begin(115200);
```

Comment:
this is cause I haven't touched arduino in forever, but are `Serial` and `Serial1` just magical global variables?

Reply:
Yep

Reply:
ack